civictechprojects/static/css/partials/_Global.scss

```diff
@@ -31,8 +31,8 @@ a[type="button"].btn-footer:active {
 // padding instead of selecting a specific height is for vertical align, but this could stand a rework later
 button,
 .btn {
-  padding: 0.4375rem 1rem; // 7px 16px; padding + border + line-height = 40px height button
-  font-weight: 600; // ask design about 500
```

Comment:
For a button label, I can change the weight to Medium instead of SemiBold. Figma shows me the value for medium style 400, Regular though regular one has the same specs in Figma. Maybe it is a design flaw.

Reply:
I tested both 500 and 600 and it's really up to you. If you want me to change the staging site to 500 so you can compare/contrast let me know, otherwise I'll use whichever value you tell me to use :)

Reply:
Yes, please, let's change to 500 :D